src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -296,6 +301,10 @@ def basic(
         else:
             helper.define_containers(ContainerType.inputs)
 
+        if readonly_inputs:
+            self.onefuzz.containers.get(readonly_inputs)
+            helper.containers[ContainerType.readonly_inputs] = readonly_inputs
+
```

Comment:
Since you want `readonly_inputs` as a container used for the fuzzing tasks, you also need to add it to this list:

https://github.com/microsoft/onefuzz/blob/6870b4c4939b18ab26cb0a7e868470d205400680/src/cli/onefuzz/templates/libfuzzer.py#L104-L108

Reply:
Updated and see the container being pulled in: INFO:onefuzz:using container: oft-readonly-inputs-<guid>